packages/std/src/math/decimal.rs

```diff
@@ -457,6 +457,12 @@ impl fmt::Display for Decimal {
     }
 }
 
+impl fmt::Debug for Decimal {
+    fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
+        write!(f, "{}", self)
```

Comment:
Okay, now this shows the value but the Debug implementation should also include the type, right? Otherwise Decimal and Decimal256 cannot be differentiated. Also https://doc.rust-lang.org/std/fmt/trait.Debug.html suggests including the type name.

What about this?

```suggestion
        write!(f, "Decimal({})", self)
```

Reply:
That makes sense, and I'd considered going in that direction originally. I didn't realize the `Debug` docs recommend including the type name, thank you for pointing that out! I've updated the PR with that change.